src/components/inputs/localized-multiline-text-input/translation-input.js

```diff
@@ -137,13 +143,7 @@ export default class TranslationInput extends React.Component {
           />
         </div>
         <div className={styles.commandsContainer}>
-          <div className={styles.commandsLeft}>
-            {this.props.error ? (
-              <div>{this.props.error}</div>
-            ) : (
-              this.props.languagesControl
-            )}
-          </div>
+          <div className={styles.commandsLeft}>{getCommands(this.props)}</div>
```

Comment:
I would turn this into:

```jsx
<div className={styles.commandsLeft}>
  {
    do {
      if (this.props.error) <div>{this.props.error}</div>
      if (this.props.warning) <div>{this.props.warning}</div>
      this.props.languagesControl;
    }
  }
</div>
```

Reply:
Is it supposed to be a do while? apparently it doesn't work as you wrote

Reply:
Sorry, I had assumed you were already familiar with `do`-expressions. They were not enabled yet for this repository (#81 was not merged). I now created & merged #87 which enables them for this repository. You need to rebase on top of master.

In case they don't work, you can always use an iife instead.

Reply:
This is the behaviour I got when I use the suggested code:

![image](https://user-images.githubusercontent.com/14938754/46005614-f06ba480-c0b5-11e8-8686-6d4093021346.png)

Using an IIFE for now

Reply:
The provided code wasn't quite correct. I pushed 0810ae0610b6e513db2bffe18b6fba18b7b73883 which adds it.

It should have been:

```js
            {do {
              if (this.props.error) <div>{this.props.error}</div>;
              else if (this.props.warning) <div>{this.props.warning}</div>;
              else this.props.languagesControl;
            }}
```